src/System.Net.Security/src/System/Net/Unix/CertModule.cs

```diff
@@ -20,7 +26,23 @@ internal override SslPolicyErrors VerifyCertificateProperties(X509Chain chain, X
                 sslPolicyErrors |= SslPolicyErrors.RemoteCertificateChainErrors;
```

Comment:
I see... so Windows has an "intermediate" state of Build failing with a valid handle.